functional-tests/fn_bridge_deposit_happy.py

```diff
@@ -25,12 +26,94 @@ def __init__(self, ctx: flexitest.InitContext):
         self.logger = get_logger("BridgeDepositHappyTest")
 
     def main(self, ctx: flexitest.RunContext):
-        return True  # this will be handled in a future PR
+        el_address_1 = "deadf001900dca3ebeefdeadf001900dca3ebeef"
+        el_address_2 = "deedf001900dca3ebeefdeadf001900dca3ebeef"
+
+        addr_1 = get_address(0)
+        addr_2 = get_address(1)
+        addr_3 = get_address(2)
+
+        # 1st deposit
+        self.test_deposit(ctx, addr_1, el_address_1)
+
+        # 2nd deposit
+        self.test_deposit(ctx, addr_2, el_address_2)
+
+        # 3rd deposit, now to a previously used address
+        self.test_deposit(ctx, addr_3, el_address_1, new_address=False)
+
+        return True
```

Comment:
This return isn't necessary either. AFAIK, flexitest only requires that the functions don't throw any exceptions.

Reply:
`ruff` complains :/

Reply:
That's strange. Didn't complain when I ran it locally.

Reply:
I get this:

```console
     ├╴  Method "main" overrides class "Test" in an incompatible manner
     │      Return type mismatch: base method returns type "Literal[True]", override returns type "None"
     │        "None" is not assignable to "Literal[True]" Pyright (reportIncompatibleMethodOverride) [28, 9]
```

Reply:
Oh, this is pyright.

Reply:
You could add an explicit `-> None` annotation for the return type of `main` and that should fix it, I think. No idea why pyright is assuming that no return type annotation means `Literal[True]`. 🤷

Reply:
This needs a change in https://codeberg.org/treyd/flexitest/src/commit/7a1483f7b9c14ee8e23003632a5b138367f19be1/flexitest/__init__.py#L35-L36

Reply:
The explicit annotation did not work either? 🫨 
If that's the case, then this can be left as is.

In `flexitest`, the function should probably just be `pass`?